CMakeLists.txt

```diff
@@ -145,7 +145,9 @@ if(WITH_ZMQ)
     find_package(ZeroMQ CONFIG REQUIRED)
   else()
     # The ZeroMQ project has provided config files since v4.2.2.
-    # TODO: Switch to find_package(ZeroMQ) at some point in the future.
+    # However, as of 2024, mainstream distributions do not yet provide
```

Comment:
I think we should just remove any dates entirely.

Reply:
Thanks! Done.